packages/spanner/system-test/spanner.js

```diff
@@ -108,6 +108,22 @@ var spanner = new Spanner(env);
       }, execAfterOperationComplete(done));
     });
 
+    describe('uneven rows', function() {
+      it('should allow inserting uneven rows', function(done) {
+        table.insert([
+          {
+            Key: generateName('id'),
+            BoolValue: true
+          },
+          {
+            Key: generateName('id'),
+            BoolValue: true,
+            IntValue: 10
+          }
+        ], done);
```

Comment:
The issue is with the source code. I think we should make it mandatory for users to specify all the values explicitly. So if something is null they should explicitly specify it as null.